feat: store key frame indices when loading video tensor from uri (#880)
docarray/docarray: docs/datatypes/video/index.md

:width: 40%
```

> Additionally, when `only_keyframes=False` the indices of the keyframes are being stored in `.tags['keyframe_indices']` when calling `.load_uri_to_video_tensor`. This way, one can easily access selected scenes: 

Reviewer: ```suggestion
Additionally, if `only_keyframes=False` the keyframe indices will be stored in `.tags['keyframe_indices']` when you call `.load_uri_to_video_tensor()`. This way, you can easily access selected scenes: 
```

> Makes sense, right?

Reviewer 2: ```suggestion
```

Not related to this PR, but there are always some people to whom things don't make sense, let's not mock them lol
